ts/src/htx.ts

```diff
@@ -1814,13 +1814,9 @@ export default class htx extends Exchange {
             const contractSize = this.safeNumber (market, 'contract_size');
             let minCost = this.safeNumber (market, 'min-order-value');
             const maxAmount = this.safeNumber (market, 'max-order-amt');
-            let minAmount = this.safeNumber (market, 'min-order-amt');
```

Comment:
@samgermain the thing is, this seems like a fallback because min-order-amt is not available 
![image](https://github.com/ccxt/ccxt/assets/43336371/2f6532d7-8a18-47a5-a44c-bf9a376b54ac)


So with your patch, we would lose the current `minAmount`

Reply:
Yeah but like https://github.com/ccxt/ccxt/issues/20580 points out, unless it's in the same units on every exchange, this is confusing

Reply:
@samgermain @carlosmiei probably for all contracts `minAmount` is 1. Now we use `contractSize` as `minAmount` for linear markets so 1 contract.